server/image_pull.go

```diff
@@ -14,11 +14,13 @@ import (
 
 	"github.com/containers/image/v5/signature"
 	imageTypes "github.com/containers/image/v5/types"
+	encconfig "github.com/containers/ocicrypt/config"
 	"github.com/cri-o/cri-o/internal/log"
 	"github.com/cri-o/cri-o/internal/storage"
 	"github.com/cri-o/cri-o/server/metrics"
 	"github.com/cri-o/cri-o/utils"
 	"github.com/docker/distribution/registry/api/errcode"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Unfortunately the linter is not happy with this, do we even need ginkgo as part of this non-test file?

Reply:
I’m sorry, that’s just me trying to diagnose test failures. I’ll mark this as draft until the tests pass.